mod/src/main/java/eu/the5zig/mod/manager/itunes/ITunesMacDelegate.java

```diff
@@ -50,7 +50,7 @@
 
 	public ITunesMacDelegate() {
 		try {
-			script = IOUtils.toString(Thread.currentThread().getContextClassLoader().getResourceAsStream("core/itunes/iTunes.applescript"));
+			script = String.valueOf(Thread.currentThread().getContextClassLoader().getResourceAsStream("core/itunes/iTunes.applescript"));
```

Comment:
Revert, String.valueOf will call InputStream#toString, which is not what we want.